source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
The branch name indicates there should be more to this than just a version bump. Where is it?